src/reynir_correct/checker.py

```diff
@@ -291,7 +293,9 @@ def annotate(sent: _Sentence) -> List[Annotation]:
                     code="E004",
                     text="Málsgreinin er sennilega ekki á íslensku",
                     detail="{0:.0f}% orða í henni finnast ekki í íslenskri orðabók"
-                        .format(words_not_in_bin/num_words * 100.0)
+                        .format(words_not_in_bin/num_words * 100.0),
+                    original="",
```

Comment:
Are these parameters required (keeping in mind that they default to `None`)?

Reply:
No, they're not needed, at first I wanted to see what exactly I was dealing with, and explicitly state the values where I could. I'll delete them where possible.